extensions/default/src/Panels/ActionButtons.tsx

```diff
@@ -2,20 +2,23 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 
-import { Button, ButtonGroup } from '@ohif/ui';
+import { LegacyButton, ButtonGroup } from '@ohif/ui';
 
 function ActionButtons({ onExportClick, onCreateReportClick }) {
   const { t } = useTranslation('MeasurementTable');
 
   return (
     <React.Fragment>
       <ButtonGroup color="black" size="inherit">
-        <Button className="px-2 py-2 text-base" onClick={onExportClick}>
+        <LegacyButton className="px-2 py-2 text-base" onClick={onExportClick}>
```

Comment:
maybe add a todo that we will revisit the button group later and that is why we are using the legacyButton here